astropy/units/tests/test_quantity.py

```diff
@@ -1391,10 +1391,6 @@ def test_quantity_fraction_power():
     assert q.dtype.kind == 'f'
 
 
-def test_inherit_docstrings():
```

Comment:
These tests now fail because we now rely on Sphinx magic, which is not tested here.

Reply:
OK, makes sense